resolwe_bio/processes/workflows/bbduk_star_featurecounts_quant.yml

```diff
@@ -0,0 +1,343 @@
+- slug: workflow-bbduk-star-fc-quant-single
+  name: BBDuk - STAR - FeatureCounts (3' mRNA-Seq, single-end)
+  data_name: "{{ reads|sample_name|default('?') }}"
+  requirements:
+    expression-engine: jinja
+  version: 1.0.0
+  type: data:workflow:quant:featurecounts:single
+  category: Pipeline
+  description: >
+    This 3' mRNA-Seq pipeline is comprised of QC, preprocessing,
+    alignment and quantification steps.
+
+    Reads are preprocessed by __BBDuk__ which removes adapters, trims
+    reads for quality from the 3'-end, and discards reads that are too short
+    after trimming. Preprocessed reads are aligned by __STAR__
+    aligner. For read-count quantification, the __FeatureCounts__ tool
+    is used.
+  input:
+    - name: reads
+      label: Input single-end reads
+      type: data:reads:fastq:single
+    - name: star_index
+      label: Star index
+      type: data:genomeindex:star
+      description: >
+        Genome index prepared by STAR aligner indexing tool.
+    - name: adapters
+      label: Adapters
+      type: list:data:seq:nucleotide
+      required: false
+      description: >
+        Provide a list of sequencing adapters files (.fasta) to be removed by BBDuk.
+    - name: annotation
+      label: Annotation
+      type: data:annotation
+    - name: stranded
+      label: Select the type of kit used for library preparation.
+      type: basic:string
+      choices:
+        - label: Strand-specific forward
+          value: forward
+        - label: Strand-specific reverse
+          value: reverse
+    - name: downsampling
+      label: Downsampling (Seqtk)
+      group:
+        - name: n_reads
+          label: Number of reads
+          type: basic:integer
+          default: 1000000
+        - name: advanced
+          label: Advanced options
+          group:
+            - name: seed
+              label: Seed
+              type: basic:integer
+              default: 11
+            - name: fraction
+              label: Fraction
+              type: basic:decimal
+              required: false
+              range: [0, 1.0]
+              description: >
+                Use the fraction of reads [0 - 1.0] from the orignal input file instead
```

Comment:
Use a comma in specification of range. Consider _Use the fraction of reads in range [0.0, 1.0] from ..._